english/bob-barker/1970s/1970s.md

```diff
@@ -0,0 +1,14 @@
+You open your eyes and see yourself reflected back in a mirror.  You have on a bright blue shirt whose collar is about twice the size it should be.  You look down and see 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.